scripts/devops_tasks/test_run_samples.py

```diff
@@ -99,7 +99,9 @@
         "mgmt_subscription_async.py",
         "mgmt_topic_async.py",
         "proxy_async.py",
-        "receive_deferred_message_queue_async.py"
+        "receive_deferred_message_queue_async.py",
+        "authenticate_using_azure_sas_credential_async.py",
+        "authenticate_using_azure_sas_credential.py"
```

Comment:
is there a reason to ignore those two? looking into the samples I think they're runnable

Reply:
skipping it for now, env variables are not ready

Reply:
could we create an issue for tracking?

Reply:
I can create one